docs/software/index.md

```diff
@@ -1,33 +1,36 @@
----
-title: Software - Overview
----
+# Overview
 
+[developing-overview]: ../development/index.md
+[lumi-p]: ../storage/parallel/lumip.md
 [easybuild]: ./installing/easybuild.md
+[spack]: ./installing/spack.md
 [contwrapper]: ./containers/wrapper.md
+[singularity-container]: ../computing/containers.md
+[singularity-jobs]: ../software/containers/cray_mpich.md
 
 ---
-This part of the documentation gives you an introduction on how to install software on LUMI. Here you can also find guides for some scientific applications supported on LUMI.
+Here you find guidance on ways to install software on LUMI as well as guides on
+using some specific software packages on LUMI. If you are looking for ways to
+optimize your software for use on LUMI, consult the [developing
+section][developing-overview] instead.
 
 ---
 
-## Learn more about installing software on LUMI
-
-On LUMI we support various ways to install software
-
- - [EasyBuild][easybuild] is our primary tool to manage the central software
-   stack and we made it easy to install additional software in your home or
-   project directory that extends that stack.
- - [Container wrapper][contwrapper] is a set of tools which wrap installations
-   inside a Apptainer/Singularity container. These tools are recommended to 
-   install Conda and pip packages on LUMI.
-
-
-## Learn more about the LUMI software Policy
+You have two options for bringing your scientific software to LUMI:
 
-If you want to learn more about the LUMI software policy, you can watch this
-recording from the last Easybuild User Meeting.
+1. Install your scientific software in your home or project directory.
+   For this we recommend using the [EasyBuild][easybuild] package manager as it
+   is used to manage the central software stack on LUMI and makes it easy to
+   install additional software that extends it. Alternatively, you may use
```

Comment:
```suggestion
   install additional software that extends it. You can find the list of softwares for which there is a currently available EasyBuild (EasyConfig) recipe for LUMI environment [here](https://github.com/Lumi-supercomputer/LUMI-SoftwareStack/tree/main/easybuild/easyconfigs) in LUMI Github repocitory. Alternatively, you may use
```

Reply:
Accepted. I might end up putting it a bit differently, but I should definitely be included.